src/components/MainMenu.js

```diff
@@ -64,13 +64,13 @@ export default class MainMenu extends PureComponent {
                     </button>
 
                     <div className={"collapse navbar-collapse" + (this.state.opened ? " show" : "")} id="navbarSupportedContent">
-                        <ul className="navbar-nav mr-auto">
-                            <li className="nav-item active"><Link to="/" className="nav-link">Úvod</Link></li>
-                            <li className="nav-item"><Link to="/schedule" className="nav-link">Rozpis</Link></li>
-                            <li className="nav-item"><a className="nav-link" href="/tabulka">Tabuľky</a></li>
-                            <li className="nav-item"><a className="nav-link" href="/playground">Ihriská</a></li>
+                        <ul className="navbar-nav w-100 d-flex ">
```

Comment:
+ thanks for adding padding, it looks better
- why did you added d-flex? its not needed